docker-compose/technical/Dockerfile-postgres

```diff
@@ -0,0 +1,3 @@
+FROM postgres:13.4
```

Comment:
I use locally Postgres 13.1
I mean it's ok to update the version we use maybe, but we should trace it into Readme (see the other comment)

Reply:
Same for others technical services (rabbit, elasctic, kibana, ....) ?